packages/eas-cli/src/vcs/clients/gitNoCommit.ts

```diff
@@ -25,4 +25,13 @@ export default class GitNoCommitClient extends GitClient {
     await ignore.initIgnoreAsync();
     return ignore.ignores(filePath);
   }
+
+  public async trackFileAsync(file: string): Promise<void> {
+    try {
+      await super.trackFileAsync(file);
+    } catch {
+      // In the no commit workflow it doesn't matter if we fail to track changes,
```

Comment:
That means that `git add` failed, I'm not aware of any cases where this could happen other than if the file is created outside of repo or if file does not exist.
We know that file exists because it was written line before, that would mean we created  plist file outside of the git repo